### ISSUE TITLE
[BUG] Delete Saved Query Not working

### ISSUE THREAD

**Issue by onwardmk** [cweijan/vscode-mysql#1172]
- Database type and version: PostgreSQL 15

When I attempt to delete a saved query, it doesn't work.

Repro Steps:

1. In Explorer, click on the trash can icon next to the saved query or right-click on it and select "Delete This Query"
2. In the confirmation prompt, select YES
3. Observe that the saved query didn't get deleted

**Comment by cweijan** [comment]
This may be a permissions issue, what is your operating system?

You can try to reveal it in finder/explorer, then try to delete it.
<img width="467" alt="image" src="https://github.com/cweijan/vscode-database-client/assets/27798227/d9914921-f04f-44e9-9ba0-ad43ebab4609">

**Comment by onwardmk** [comment]
My host OS is Windows 11, but for development work, I launch vscode from WSL 2:

Here is the output from `wsl --version`:
```
WSL version: 2.1.5.0
Kernel version: 5.15.146.1-2
WSLg version: 1.0.60
MSRDC version: 1.2.5105
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22631.3672
```

The distro I'm using is Ubuntu 22.04.

**Comment by cweijan** [comment]
Fixed in version 7.4.5.
